bin/packager/packager.js

```diff
@@ -0,0 +1,75 @@
+const packager = require('electron-packager');
+const setLanguages = require('electron-packager-languages');
+const path = require('path');
+
+const { OS } = process.env;
```

Comment:
I originally used that, but went against it because it would prevent me from building for other OSs that aren't my own. I regularly test, from Linux, if I can package for Windows and macOS